index.php

```diff
@@ -167,6 +167,11 @@ function initMap() {
     </div>
   </section><!-- End Hero -->
 
+  <!-- Welcome Tooltip -->
+  <div id="welcome-tooltip">
+      Welcome Admin! <!--Username goes here -->
```

Comment:
@anand-nakat Please make this dynamic, so that the name of the signed in user is displayed.
You can use the php username variable here.